vignettes/R-dev-blog-post.Rmd

```diff
@@ -115,12 +119,12 @@ inherits(x, tensorflow::tf$Variable)
 
 ### `@`
 
-Prior to R 4.3.0, `@` could be used only with S4 objects.
+Prior to R 4.3.0, `@` could be used to access slots of S4 objects.
 Beginning with R 4.3.0, this operator can now perform dispatch S3 dispatch as well.
```

Comment:
the first `dispatch `  should be deleted here (was there already)

Reply:
Thanks, fixed now.